Jenkinsfile

```diff
@@ -1,90 +1,72 @@
 #!groovy
 
-// import shared library: https://github.com/cmu-delphi/jenkins-shared-library
+// Import shared lib.
 @Library('jenkins-shared-library') _
 
-pipeline {
+/*
+   Declare variables.
+   - indicator_list should contain all the indicators to handle in the pipeline.
+   - Keep in sync with '.github/workflows/python-ci.yml'.
+   - TODO: Get this list automatically from python-ci.yml at runtime?
```

Comment:
```suggestion
   - TODO: #527 Get this list automatically from python-ci.yml at runtime?
```

Reply:
I think I am doing this part wrong maybe. I created the issue, then updated the line and pushed like:

<img width="625" alt="image" src="https://user-images.githubusercontent.com/3670389/99297347-24662e80-2816-11eb-960b-050158a3994f.png">

I guess I'd expected it to update here in some way (I guess it shows 'Outdated' because I'd updated it outside of this conversation), but what is the best workflow?

Reply:
Oh, I can't accept it now anyway, probably because of my earlier commit. Hmm...

Reply:
oh that's weird. yes if you push into the branch the PR normally updates. Maybe we passed in the night?